dotnet/src/SemanticKernel.Abstractions/Contents/ImageContent.cs

```diff
@@ -17,9 +17,14 @@ public sealed class ImageContent : KernelContent
     public Uri? Uri { get; set; }
 
     /// <summary>
-    /// The image binary data.
+    /// The image media type.
     /// </summary>
-    public BinaryData? Data { get; set; }
+    public string? MediaType { get; set; }
```

Comment:
You introduced MediaType for ImageContent but why not for AudioContent also?

Reply:
Make sense, added.